### PR TITLE
Reduce netdatacli size

### PR DESCRIPTION
**Pull request by stelfrag**
##### Summary
Reduce the netdatacli binary size

### CODE REVIEW

File: cli/cli.c [hunk 1]
```diff
@@ -174,7 +223,7 @@ int main(int argc, char **argv)
         size_t to_copy;
 
         to_copy = MIN(strlen(argv[i]), MAX_COMMAND_LENGTH - 1 - command_string_size);
-        strncpyz(command_string + command_string_size, argv[i], to_copy);
```
Comment (ktsaou) [inline cli/cli.c:177]: this may not be null terminated

**Comment by underhood** [comment]
size went to 121K from 1,5M.  nice

**Review by underhood — DISMISSED** [review]

**Comment by thiagoftsm** [comment]
Hello @stelfrag ,

After to clone this branch, I observed the following errors during compilation time:

```sh
proto/agent/v1/cmds.proto:7:1: warning: Import proto/aclk/v1/lib.proto is unused.
./libnetdata/libnetdata.h:513:35: error: expected declaration specifiers or '...' before string constant
  513 | #define freez(ptr) freez_int(ptr, __FILE__, __FUNCTION__, __LINE__)
      |                                   ^~~~~~~~
cli/cli.c:29:6: note: in expansion of macro 'freez'
   29 | void freez(void *ptr) {
      |      ^~~~~
In file included from ./daemon/common.h:6,
                 from cli/cli.h:6,
                 from cli/cli.c:3:
./libnetdata/libnetdata.h:513:45: error: expected declaration specifiers or '...' before '__FUNCTION__'
  513 | #define freez(ptr) freez_int(ptr, __FILE__, __FUNCTION__, __LINE__)
      |                                             ^~~~~~~~~~~~
cli/cli.c:29:6: note: in expansion of macro 'freez'
   29 | void freez(void *ptr) {
      |      ^~~~~
./libnetdata/libnetdata.h:513:59: error: expected declaration specifiers or '...' before numeric constant
  513 | #define freez(ptr) freez_int(ptr, __FILE__, __FUNCTION__, __LINE__)
      |                                                           ^~~~~~~~
cli/cli.c:29:6: note: in expansion of macro 'freez'
   29 | void freez(void *ptr) {
      |      ^~~~~
./libnetdata/libnetdata.h:511:41: error: expected declaration specifiers or '...' before string constant
  511 | #define mallocz(size) mallocz_int(size, __FILE__, __FUNCTION__, __LINE__)
      |                                         ^~~~~~~~
cli/cli.c:33:7: note: in expansion of macro 'mallocz'
   33 | void *mallocz(size_t size) {
      |       ^~~~~~~
./libnetdata/libnetdata.h:511:51: error: expected declaration specifiers or '...' before '__FUNCTION__'
  511 | #define mallocz(size) mallocz_int(size, __FILE__, __FUNCTION__, __LINE__)
      |                                                   ^~~~~~~~~~~~
cli/cli.c:33:7: note: in expansion of macro 'mallocz'
   33 | void *mallocz(size_t size) {
      |       ^~~~~~~
./libnetdata/libnetdata.h:511:65: error: expected declaration specifiers or '...' before numeric constant
  511 | #define mallocz(size) mallocz_int(size, __FILE__, __FUNCTION__, __LINE__)
      |                                                                 ^~~~~~~~
cli/cli.c:33:7: note: in expansion of macro 'mallocz'
   33 | void *mallocz(size_t size) {
      |       ^~~~~~~
./libnetdata/libnetdata.h:510:55: error: expected declaration specifiers or '...' before string constant
  510 | #define callocz(nmemb, size) callocz_int(nmemb, size, __FILE__, __FUNCTION__, __LINE__)
      |                                                       ^~~~~~~~
cli/cli.c:42:7: note: in expansion of macro 'callocz'
   42 | void *callocz(size_t nmemb, size_t size) {
      |       ^~~~~~~
./libnetdata/libnetdata.h:510:65: error: expected declaration specifiers or '...' before '__FUNCTION__'
  510 | #define callocz(nmemb, size) callocz_int(nmemb, size, __FILE__, __FUNCTION__, __LINE__)
      |                                                                 ^~~~~~~~~~~~
cli/cli.c:42:7: note: in expansion of macro 'callocz'
   42 | void *callocz(size_t nmemb, size_t size) {
      |       ^~~~~~~
./libnetdata/libnetdata.h:510:79: error: expected declaration specifiers or '...' before numeric constant
  510 | #define callocz(nmemb, size) callocz_int(nmemb, size, __FILE__, __FUNCTION__, __LINE__)
      |                                                                               ^~~~~~~~
cli/cli.c:42:7: note: in expansion of macro 'callocz'
   42 | void *callocz(size_t nmemb, size_t size) {
      |       ^~~~~~~
./libnetdata/libnetdata.h:512:53: error: expected declaration specifiers or '...' before string constant
  512 | #define reallocz(ptr, size) reallocz_int(ptr, size, __FILE__, __FUNCTION__, __LINE__)
      |                                                     ^~~~~~~~
cli/cli.c:51:7: note: in expansion of macro 'reallocz'
   51 | void *reallocz(void *ptr, size_t size) {
      |       ^~~~~~~~
./libnetdata/libnetdata.h:512:63: error: expected declaration specifiers or '...' before '__FUNCTION__'
  512 | #define reallocz(ptr, size) reallocz_int(ptr, size, __FILE__, __FUNCTION__, __LINE__)
      |                                                               ^~~~~~~~~~~~
cli/cli.c:51:7: note: in expansion of macro 'reallocz'
   51 | void *reallocz(void *ptr, size_t size) {
      |       ^~~~~~~~
./libnetdata/libnetdata.h:512:77: error: expected declaration specifiers or '...' before numeric constant
  512 | #define reallocz(ptr, size) reallocz_int(ptr, size, __FILE__, __FUNCTION__, __LINE__)
      |                                                                             ^~~~~~~~
cli/cli.c:51:7: note: in expansion of macro 'reallocz'
   51 | void *reallocz(void *ptr, size_t size) {
```

I compiled netdata with the following `CFLAGS`:

```sh
CFLAGS="-Og -ggdb -Wall -Wextra -fno-omit-frame-pointer -Wformat-signedness -fstack-protector-all -Wformat-truncation=2 -Wunused-result -DENABLE_HTTPD=1 -DHAVE_BACKTRACE=1 -DNETDATA_TRACE_ALLOCATIONS=1 -DNETDATA_DEV_MODE=1 -DNETDATA_VERIFY_LOCKS=1 -DMQTT_WSS_DEBUG=1"
```

**Comment by stelfrag** [comment]
> After to clone this branch, I observed the following errors during compilation time:

Thanks @thiagoftsm, compilation errors fixed -- please check

**Review by thiagoftsm — APPROVED** [review]
After last commit everything works as expected, LGTM!

**Review by underhood — APPROVED** [review]

**Pull request merged**